src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -497,16 +494,18 @@ class SocialSecurityObserver(BaseObserver):
 
     ADDITIONAL_INPUT_COLUMNS = ["tracked", "alive", "entrance_time", "exit_time", "has_ssn"]
     OUTPUT_COLUMNS = [
+        "household_id",
         "first_name_id",
         "middle_name_id",
         "last_name_id",
         "date_of_birth",
-        "event_type",
-        "event_date",
         "sex",
         "race_ethnicity",
+        "event_type",
+        "event_date",
```

Comment:
Is there a reason for the reordering?

Reply:
Oh, I forgot to delete the now-unued POST_PROCESSING_FIRST_NAME_METADATA_COLS which I think will make it more clear; see the new commit. I basically just re-ordered to match that.